modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/identity/mgt/AccountLockEnabledTestCase.java

```diff
@@ -153,17 +183,53 @@ public void testSuccessfulEmailTemplateRetrievalAccountUnLock() throws Exception
                     "Test Failure : Email Content applicable for account unlock is not available.");
     }
 
+    @SetEnvironment(executionEnvironments = {ExecutionEnvironment.ALL})
+    @Test(groups = "wso2.is", description = "Check whether the secondary user-store user account lock successfully")
+    public void testSuccessfulLockedSecondaryUserStoreUser() {
```

Comment:
Let's use DataProvider to pass the user creation details of both Primary userstore domain user and Secondary userstore domain user to the testSuccessfulLockedInitially() method and remove this method